lib/chain.js

```diff
@@ -52,6 +53,10 @@ module.exports = class SetupChain {
     }
   }
 
+  get ctx() {
```

Comment:
nit: `current_context` or `active_context` or something like that?

Reply:
I flipped them, so `context` is the public-ish facing property.